api/app/controllers/spree/api/v2/base_controller.rb

```diff
@@ -82,6 +82,8 @@ def spree_current_user
           @spree_current_user ||= doorkeeper_token.resource_owner
         end
 
+        alias try_spree_current_user spree_current_user  # for compatibility with spree_legacy_frontend
```

Comment:
I see that this line fixes tests in legacy frontend, but won't it also end up overriding the original `try_spree_current_user` method? See: https://github.com/spree/spree/blob/768a80cee383165ea208bede324213a064072c38/core/lib/spree/core/controller_helpers/auth.rb#L61-L63

Based on the comment on this method, I'm guessing that there's some code where `current_spree_user` needs to be used instead of `spree_current_user`.

Reply:
As far as I know it shouldn't becouse the alias is for api which doesn't import this auth controller helper and vice versa -> nothing in core imports Api::V2::BaseController.
And this line actually fixes tests of api,  the line which fixes legacy_frontend is in locale.rb. Altough I've just realised that I missed changing spree_current_user to try_spree_current_user in one place.

Reply:
Awesome, yeah that makes sense!